src/vcRender.cpp

```diff
@@ -1053,13 +1060,14 @@ void vcRender_TransparentPass(vcState *pProgramState, vcRenderContext *pRenderCo
 
     for (size_t i = 0; i < renderData.images.length; ++i)
     {
-      double zScale = 1.0;
-      zScale -= udMag3(pProgramState->camera.position - renderData.images[i]->position) / pProgramState->settings.presentation.imageRescaleDistance;
+      vcImageRenderInfo *pImage = renderData.images[i];
+      double distToCamera = udMag3(pProgramState->camera.position - renderData.images[i]->position);
```

Comment:
This should be `pImage` otherwise it doesn't really make sense to have `pImage` grabbed here just for the last line, that might not be hit.